### ISSUE TITLE
TV Show Parser Renaming Not Using Custom Show Name

### ISSUE THREAD

**Issue by psouza4** [psouza4/mediacentermaster#1448]
Originally reported on Google Code with ID 1451
```
Hi Pete,

My metadata shows:
Show Name: "Beauty and the Beast"
Search Term: "beauty beast 2012"
tvdb ID: "258959"
imdb ID: "tt2193041"

When I parse an episode I get something like this:

8:58 PM :: MCM:  Now working on beauty.and.the.beast.2012.s02e01.720p.hdtv.x264-2hd.mkv...
8:58 PM :: MCM:  Automatically detecting beauty.and.the.beast.2012.s02e01.720p.hdtv.x264-2hd.mkv...
8:58 PM :: MCM:      D:\Downloads\TV Downloads\Beauty.and.the.Beast.2012.S02E01.720p.HDTV.x264-2HD
[PublicHD]\beauty.and.the.beast.2012.s02e01.720p.hdtv.x264-2hd.mkv
8:58 PM :: MCM:  Automatically detecting Beauty.and.the.Beast.2012.S02E01.720p.HDTV.x264-2HD
[PublicHD]...
8:58 PM :: MCM:      (parent folder or archive detection attempt)
8:58 PM :: thetvdb.com  Searching for: Beauty And The Beast 2012
8:59 PM :: thetvdb.com  Found series ID: 258959
8:59 PM :: thetvdb.com  Searching by season 2, episode 1
8:59 PM :: thetvdb.com  Loading existing metadata for show ID #258959
8:59 PM :: thetvdb.com  Fetching metadata for season 2, episode 1
8:59 PM :: MCM:  ... this is a T.V. show episode  Beauty and the Beast 2012 - 2x01
- "Who Am I?"
8:59 PM :: thetvdb.com  Searching by season 2, episode 1
8:59 PM :: MCM:  ... this is a T.V. show episode (detected from parent folder)
8:59 PM :: MCM:  Moving and renaming to:
8:59 PM :: MCM:      I:\HD TV Shows\Beauty and the Beast\Season 2
8:59 PM :: MCM:      Beauty and the Beast (2012) - 2x01 - Who Am I.mkv
8:59 PM :: MCM:  Fetching metadata for the new episode...
8:59 PM :: -----------------------------------------------------------------------------------------

Now my issue is that my show name "Beauty and the Beast" but the parser renames the
show as "Beauty and the Beast (2009)".
I expect: Beauty and the Beast - 2x01 - Who Am I.mkv
I get:    Beauty and the Beast (2012) - 2x01 - Who Am I.mkv

Can please add in an option (or change the functionality) to use the show name as defined
in the metadata rather than the one pulled down from tvdb?

Thanks,
-Chris
```

Reported by `tobeychris` on 2013-11-19 18:16:56

**Comment by MikeFalcor** [comment]
And this is **still** an issue...

Can we get an update?

**Comment by psouza4** [comment]
Seems that some of the more recent comments have confused the actual issue... the comment that doug added (prior to importing to GitHub -- it looks like my comment, but it's his, quoting me at the forums) seems to be related to something else entirely.

The next version of Media Center Master will display and rename with the year in the title, as expected.  You will need to refresh those titles after the update to clear out the cache (or you can do this in bulk by using the Ctrl+F5 option from the Media menu).

**Comment by psouza4** [comment]
The download parser and media list (as well as other features that display a TV show's title) will now properly display the year as part of the title.

Fixed and waiting for next release.

**Comment by psouza4** [comment]
Released in version 2.15 build 6616.574 (2.15.6616.574).
